### PR TITLE
fix(tooltip): Tooltip textStyle lineHeight does not work in default 

### PR DESCRIPTION
**Pull request by sz-p**
## Brief Information

This pull request is in the type of:

- [x] bug fixing
- [ ] new feature
- [ ] others

### Fixed issues

#20391 

### Before: What was the problem?

lineHeight not work

### After: How does it behave after the fixing?

add line-height in css

## Document Info

One of the following should be checked.

- [x] This PR doesn't relate to document changes
- [ ] The document should be updated later
- [ ] The document changes have been made in apache/echarts-doc#xxx

### ZRender Changes

- [ ] This PR depends on ZRender changes (ecomfe/zrender#xxx).

### Related test cases or examples to use the new APIs

N.A.

### Merging options

- [ ] Please squash the commits into a single one when merging.

### CODE REVIEW

File: src/component/tooltip/TooltipHTMLContent.ts [hunk 1]
```diff
@@ -151,9 +151,11 @@ function assembleFont(textStyleModel: Model<TooltipOption['textStyle']>): string
 
     cssText.push('font:' + textStyleModel.getFont());
 
+    // @ts-ignore, leave it to the tooltip refactor.
+    const lineHeight = textStyleModel.get('lineHeight') ?? Math.round(fontSize * 3 / 2);
```
Comment (Ovilia) [inline src/component/tooltip/TooltipHTMLContent.ts:155]: Use `zrUtil.retrive2` instead.

File: src/component/tooltip/tooltipMarkup.ts [hunk 2]
```diff
@@ -42,6 +42,17 @@ type TextStyle = string | RichTextStyle;
 
 const TOOLTIP_LINE_HEIGHT_CSS = 'line-height:1';
 
+function getTooltipLineHeight(
+    textStyle: TooltipOption['textStyle']
+) {
+    const lineHeight = textStyle.lineHeight;
+    if (lineHeight === undefined) {
```
Comment (Ovilia) [inline src/component/tooltip/tooltipMarkup.ts:49]: Use `== null` to check undefined or null values.

**Review by Ovilia — CHANGES_REQUESTED** [review]
Thanks for your contribution. It seems Okay to me only with a few small changes.

**Comment by sz-p** [comment]
Done

**Review by Ovilia — CHANGES_REQUESTED** [review]

File: src/component/tooltip/tooltipMarkup.ts [hunk 3]
```diff
@@ -46,7 +46,7 @@ function getTooltipLineHeight(
     textStyle: TooltipOption['textStyle']
 ) {
     const lineHeight = textStyle.lineHeight;
-    if (lineHeight === undefined) {
+    if (lineHeight === undefined || lineHeight === null) {
```
[the remark this answers is not in the document]
Comment (sz-p) [reply src/component/tooltip/tooltipMarkup.ts]: OK, that's work well

**Review by Ovilia — APPROVED** [review]

**Pull request merged**